learn/advanced/sorting.md

```diff
@@ -17,7 +17,7 @@ To allow your users to sort results at search time you must:
 3. Update Meilisearch's [ranking rules](/learn/core_concepts/relevancy.md) (optional)
 
 ::: note
-Meilisearch sorts strings in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.
+Documents used for sort are converted to lowercase and then sorted in lexicographic order based on their byte values. For example, `á`, which has a value of 225, will be sorted after `z`, which has a value of 122.
```

Comment:
For me this wording raises the question of whether the fields in the returned documents will actually be lowercased. I'm guessing that they're only lowercased for the purposes of sorting and will be unchanged in the results. If so, could you rephrase to make that clear?